external/webserver/http_client.c

```diff
@@ -1163,36 +1201,26 @@ int http_send_response_helper(struct http_client_t *client, int status, const ch
 		// Append extra CRLF to mark headers done
 		buflen += snprintf(buf + buflen,
 					HTTP_CONF_MAX_REQUEST_LENGTH - buflen, "\r\n");
-
 		// Include response body
 		if (body) {
-			buflen += snprintf(buf + buflen,
-						HTTP_CONF_MAX_REQUEST_LENGTH - buflen,
-						"%s",
-						body);
+			memcpy(buf + buflen, body, HTTP_CONF_MAX_REQUEST_LENGTH - buflen);
```

Comment:
How about declaring a local variable and set as `HTTP_CONF_MAX_REQUEST_LENGTH - buflen`?
`HTTP_CONF_MAX_REQUEST_LENGTH - buflen` is used multiple times, so we can reduce the code and improve the performance.

Reply:
not required